pyalgostrategypool/mean_reversion_bollinger_bands_v2.py

```diff
@@ -0,0 +1,70 @@
+import talib
+from pyalgotrading.constants import *
+from pyalgotrading.strategy import StrategyBase
+
+
+class MeanReversionBollingerBandsV2(StrategyBase):
+    name = 'Mean Reversion Bollinger Bands V2'
```

Comment:
if there is no V1, remove the V2 from here

Reply:
Done